action.yml

```diff
@@ -52,7 +63,7 @@ runs:
     - name: Run PMD scan against changes
       run: |
         ${{ github.action_path }}/delta-scan.sh \
-          "pmd-ruleset/pmd-ruleset.xml" \
+          "pmd-ruleset/${{ inputs.pmd-ruleset-path }}" \
```

Comment:
I'm confused with the comma-separated paths - multiple rule set paths should be resolved like: `pmd-ruleset/pmd-ruleset1.xml,pmd-ruleset2.xml`?

Reply:
Yes - if your project root contains a ruleset called pmd-ruleset2.xml then this is exactly the syntax.

Reply:
for consistency is probably better to not hardcode `pmd-ruleset/` here and add it to the `default` of `pmd-ruleset-path` input

Reply:
Won't the hardcoded `pmd-ruleset/` be a problem if someone will set custom `pmd-ruleset-repo`?

Reply:
Ah - you're right, but unfortunately the support for multiple rulesets is a bit of a hack. This supports one remote ruleset and any number of additional local rulesets. The remote ruleset is cloned into a directory called `pmd-ruleset` (hardcoded in the checkout action) and the local rulesets are the responsibility of the calling workflow.

Reply:
> Won't the hardcoded `pmd-ruleset/` be a problem if someone will set custom `pmd-ruleset-repo`?

No - this corresponds to the target directory in the checkout@v3 action. It doesn't matter what the original ruleset repo is called.